tests/components/generic/test_camera.py

```diff
@@ -63,10 +66,86 @@ async def test_fetching_url(
     body = await resp.read()
     assert body == fakeimgbytes_png
 
+    # sleep .1 seconds to make cached image expire
+    await asyncio.sleep(0.1)
```

Comment:
I guess it's fine to just sleep 100ms here instead of switching to freezegun for these tests...?